src/Sample.Extension/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.WebJobs" version="1.2.0-alpha-10306" targetFramework="net45" />
-  <package id="Microsoft.Azure.WebJobs.Core" version="1.2.0-alpha-10306" targetFramework="net45" />
+  <package id="Microsoft.Azure.WebJobs" version="2.0.0-alpha-10324" targetFramework="net45" />
```

Comment:
Did you update the nuspecs for the packages to declare that we require this new version?